packages/create-pkg-cli/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "create-pkg",
+  "version": "0.0.0",
+  "description": "Scaffold a package",
+  "author": "Wes Todd <wes@wesleytodd.com>",
+  "keywords": [
+    "scaffold",
+    "package",
+    "git",
+    "module",
+    "package.json",
+    "readme",
+    "mocha",
+    "standard"
+  ],
+  "license": "ISC",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:pkgjs/create.git"
+  },
+  "bin": {
+    "create-package": "bin/create-package"
+  },
+  "scripts": {
+    "test": "standard && mocha --timeout=60000",
+    "test:debug": "mocha --inspect --inspect-brk --timeout=0",
+    "release": "npm t && standard-version && npm publish",
+    "postpublish": "git push origin && git push origin --tags"
+  },
+  "dependencies": {
+    "cptmpl": "0.0.5",
+    "create-git": "^1.0.0-2",
+    "create-package-json": "^1.0.0-2",
+    "loggerr": "^3.0.0-3",
+    "opta": "0.0.6",
+    "semver": "^7.3.2"
+  },
+  "devDependencies": {
+    "fs-extra": "^8.0.1",
+    "fs-test-fixtures": "^0.1.3",
+    "mocha": "^6.2.2",
+    "standard": "^14.3.1",
+    "standard-version": "^9.0.0"
+  },
+  "support": true
```

Comment:
Was this added in this PR? Not sure it matters, but I just didn't remember it being there in the old one. Also, should be we a bit more particular about our support and use a bit more of the spec?

Reply:
Yeah not sure how that slipped in. It might have been in there already but wasn't known until i moved stuff around. We can definitely add more to the support spec for sure.

Reply:
Doesn't need to block this PR, especially since support for that field is basically non-existent lol. Just something I noticed.